backend/Services/CsvReader.php

```diff
@@ -5,14 +5,21 @@
 namespace Internships\Services;
 
 use Exception;
+use Internships\FileSystem\DirectoryManager;
 
 class CsvReader
 {
     public const CSV_READ_LENGTH = 0;
     public const CSV_SEPARATOR = ",";
 
-    public function getCSVData(string $fullPath, array $fieldDefines): array
+    public function __construct(
+        protected DirectoryManager $directoryManager
+    ) {
+    }
+
+    public function getCSVData(string $resourceRelativePath, string $fileName, array $fieldDefines): array
```

Comment:
What's `$fieldDefines`?

Reply:
It's array of fields within DataFactory. It's used to verify if current entry has same amount of elements. 

It could probably be moved to integrity test.